mods/e2140/content/ed/buildings/bio_center/rules.yaml

```
@@ -0,0 +1,87 @@
+ed_buildings_bio_center:
+	Inherits: ^CoreBuilding
+	Tooltip:
+		Name: Bionic Production Center
```

Similar to Robot Factory, please use original name: *Bio-Center* (with or without hyphen).

```suggestion
		Name: Bio-Center
```